<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutDownForm</class>
 <widget class="QDialog" name="ShutDownForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>232</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>232</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выключение ПК</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioPower">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выключить</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/ico/gnome-shutdown.png</normaloff>:/ico/gnome-shutdown.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioReset">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Перезагрузить</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/ico/windowsrestart_7374.png</normaloff>:/ico/windowsrestart_7374.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioLogout">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Завершить сеанс</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/ico/gnome-logout_5442.png</normaloff>:/ico/gnome-logout_5442.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>192</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>40</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
